man/systemd.exec.xml

```diff
@@ -738,6 +738,37 @@
       </varlistentry>
 
       <varlistentry>
+        <term><varname>LogLevelMax=</varname></term>
+        <listitem><para>Configures filtering by log level of log messages generated by this unit. Takes a
+        <command>syslog</command> log level, one of <option>emerg</option> (lowest), <option>alert</option>,
+        <option>crit</option>, <option>err</option>, <option>warning</option>, <option>notice</option>,
+        <option>info</option>, <option>debug</option> (highest). See <citerefentry
+        project='man-pages'><refentrytitle>syslog</refentrytitle><manvolnum>3</manvolnum></citerefentry> for
+        details. By default no filtering is applied (i.e. the default maximum log level is <option>debug</option>). Use
+        this option to configure the logging system to drop log messages of a specific service above the specified
+        level. For example, set <varname>LogLevelMax=</varname><option>info</option> in order to turn off debug logging
+        of a particularly chatty unit. Note that the the configured level is applied to any log messages written by any
+        of the processes belonging to this unit, sent via any supported logging protocol. The filtering is applied
+        early in the logging pipeline, before any kind of further processing is done. Moreover, messages which pass
+        through this filter successfully might still be dropped by filters applied at a later stage in the logging
+        subsystem. For example, <varname>MaxLevelStore=</varname> configured in
+        <citerefentry><refentrytitle>journald.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry> might
+        prohibit messages of higher log levels to be stored on disk, even though the per-unit
+        <varname>LogLevelMax=</varname> permitted it to be processed.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>LogExtraField=</varname></term>
+        <listitem><para>Configures an additional log metadata field to include in any log record generated by processes
+        associated with this unit. Takes journal field specifications in the format <literal>FIELD=VALUE</literal>, see
+        <citerefentry><refentrytitle>systemd.journal-fields</refentrytitle><manvolnum>7</manvolnum></citerefentry> for
+        details on the journal field concept. Event though the underlying journal implementation permits binary field
+        values, this setting accepts only valid UTF-8 values. Note that this setting not only is useful to attach
+        additional metadata to log records of a unit, but given that all fields and values are indexed may also be used
+        to implement cross-unit log record matching.</para></listitem>
```

Comment:
Hmm, let's talk about syntax. Shouldn't this be exactly the same as `Environment=`? It seems a bit surprising to have this take only a single field on a line, and to not allow resetting the list. I think that for consistency it'd be good to make this `LogExtraFields=` and allow quoted assignments as elsewhere.

Reply:
true. will rearrange this. We do allow ressetting the list already btw. I just forgot to document it.